envs/example/defaults.yml

```diff
@@ -322,15 +323,15 @@ keystone:
     - name: heat
       type: orchestration
       description: 'Heat Orchestration API'
-      public_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      internal_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
-      admin_url: https://{{ endpoints.main }}:8005/v1/%(tenant_id)s
+      public_url: https://{{ endpoints.main }}:8004/v1/%(tenant_id)s
```

Comment:
For #1144, it has fixed all supporting components since it has to test the functions after making the changes. Heat is not part of it since It is difficult to test heat at the moment. So we decided to include the heat port changes in this request since we have to test it any way so we just tested with new port changes. It will be the same for ceilomenter that new components will need to have the right ports configured.